kerastuner/engine/tuner_utils.py

```diff
@@ -94,17 +98,65 @@ def on_epoch_end(self, epoch, logs=None):
 # TODO: Add more extensive display.
 class Display(object):
 
-    def __init__(self, verbose=1):
+    def __init__(self, oracle, verbose=1):
         self.verbose = verbose
+        self.oracle = oracle
 
     def on_trial_begin(self, trial):
         if self.verbose >= 1:
-            display.section('Starting new trial')
+            print()
+            trial_number = self.oracle.get_trial_number(trial)
+            total_trials = self.oracle.max_trials or '?'
+            print('Search: Running Trial {}/{}'.format(trial_number, total_trials))
```

Comment:
Let's just show the number of Trials this worker has complete, by keeping a running tally in tuner_utils.Display